plugins/cilium-cni/04-flannel-cilium-cni.conflist

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Is there a specific order this file needs to be in the CNI conf directory? Or does it just have to be before `x-cilium.conf`?